package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "hapi-university",
+  "license": "MIT",
+  "version": "0.0.1",
+  "description": "Hapi.js University",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:averyvery/university.git"
+  },
+  "scripts": {
+    "start": "nodemon lib/index.js --exec babel-node",
+    "test": "babel-node ./node_modules/istanbul/lib/cli cover _mocha && open coverage/lcov-report/index.html"
+  },
+  "dependencies": {
+    "babel-cli": "6.3.17",
+    "babel-preset-es2015": "6.3.13",
+    "hapi": "11.1.3",
+    "nodemon": "1.8.1"
+  },
+  "devDependencies": {
+    "istanbul": "1.0.0-alpha.2",
```

Comment:
istanbul alpha seems to handle compiled JS very well

![screen shot 2015-12-23 at 6 27 01 pm](https://cloud.githubusercontent.com/assets/72114/11987826/6a495f7c-a9a3-11e5-85e5-4fa76de78272.png)